src/apps/user/models.py

```diff
@@ -0,0 +1,16 @@
+from django.contrib.auth.models import AbstractUser
+from django.db import models
+
+
+class User(AbstractUser):
+    class UserType(models.TextChoices):
```

Comment:
Maybe it will be better to create OneToOne Profile model?

Reply:
Created ForeignKey instead.